recipes/IWSLT22_lowresource/AST/transformer/README.md

```diff
@@ -79,6 +95,7 @@ One should change hparams/train_samu_mbart_st.yaml to hparams/train_samu_nllb_st
 | 3 | train_w2v2_nllb_st.yaml | 11.09 | 8.70 | [DropBox](https://www.dropbox.com/sh/spp2ijgfdbzuz26/AABkJ97e72D7aKzNLTm1qmWEa?dl=0) |
 | 4 | train_samu_mbart_st.yaml | 13.41 | 10.28 | [DropBox](https://www.dropbox.com/sh/98s1xyc3chreaw6/AABom3FnwY5SsIvg4en9tWC2a?dl=0) |
 | 5 | train_samu_nllb_st.yaml | 13.89 | 11.32 | [DropBox](https://www.dropbox.com/sh/ekkpl9c3kxsgllj/AABa0q2LrJe_o7JF-TTbfxZ-a?dl=0) |
+| 6 | train_speecht5_st.yaml | 6.00 | 5.28 | [DropBox](https://www.dropbox.com/scl/fo/q5zx8ah7rzeoz0fg6ea62/h?rlkey=y68eo4faog0nz4t9c4lyxoh4x&dl=0) |
```

Comment:
btw, about the results, I am not that familiar with the field of speech translation, but, what do you think of these results? Are they expected given SpeechT5 or were you expecting better results ? (from my very shallow understanding, higher bleu is better right?)

Reply:
Ok got it!